project-base/tests/ReadModelBundle/Functional/Image/ImageViewRepositoryTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ReadModelBundle\Functional\Image;
+
+use Shopsys\FrameworkBundle\Model\Product\Product;
+use Shopsys\ReadModelBundle\Image\ImageViewInterface;
+use Shopsys\ReadModelBundle\Image\ImageViewRepository;
+use Tests\ShopBundle\Test\FunctionalTestCase;
+
+class ImageViewRepositoryTest extends FunctionalTestCase
+{
+    public function testGetForSingleEntityId(): void
+    {
+        $testedProductId = 1;
```

Comment:
I'm not sure, but I think you should use

```php
$product = $this->getReference(ProductDataFixture::PRODUCT_PREFIX . '1');
$productId = $product->getId();
```

this deserves consultation with someone else @vitek-rostislav ?

Reply:
Hi, I think it is not necessary here - we do not need the whole entity, just _an_ ID. Maybe constant would be more suitable, however, I am ok with the current state

Reply:
It's in the constant now